exchange/exchange.go

```diff
@@ -71,6 +72,7 @@ func NewExchange(client *http.Client, cache prebid_cache_client.Client, cfg *con
 	e.currencyConverter = currencyConverter
 	e.UsersyncIfAmbiguous = cfg.GDPR.UsersyncIfAmbiguous
 	e.defaultTTLs = cfg.CacheURL.DefaultTTLs
+	e.enforceCCPA = cfg.CCPA.Enforce
 	return e
```

Comment:
I realize this change would imply a thorough refactoring effort that is out of the scope of this PR so we don't have to do it right now, but I believe we could be better off if we consolidate all the privacy enforcement in a single class with its own fields and methods
```
 40   type exchange struct {
 41       adapterMap          map[openrtb_ext.BidderName]adaptedBidder
 42       me                  pbsmetrics.MetricsEngine
 43       cache               prebid_cache_client.Client
 44       cacheTime           time.Duration
 45 -     gDPR                gdpr.Permissions
    +     privacyPolicies     policiesAndPermissions
 46       currencyConverter   *currencies.RateConverter
 47       UsersyncIfAmbiguous bool
 48       defaultTTLs         config.DefaultTTLs
 49 -     enforceCCPA         bool
 50   }
 51
 52   --- 11 lines: Container to pass out response ext data from the GetAllBids goroutines back into the main thread-----------
 63
 64   func NewExchange(client *http.Client, cache prebid_cache_client.Client, cfg *config.Configuration, metricsEngine pbsmetrics.MetricsEngine, infos adapters.BidderInfos, gDPR gdpr.Permis>
 65       e := new(exchange)
 66
 67       e.adapterMap = newAdapterMap(client, cfg, infos)
 68       e.cache = cache
 69       e.cacheTime = time.Duration(cfg.CacheURL.ExpectedTimeMillis) * time.Millisecond
 70       e.me = metricsEngine
 71 -     e.gDPR = gDPR
    +     e.privacyPolicies = &policiesAndPermissions{GDPRPermissions:GDPR,CCPA:&policy.CCPA{enforceCCPA:cfg.CCPA.Enforce}
 72       e.currencyConverter = currencyConverter
 73       e.UsersyncIfAmbiguous = cfg.GDPR.UsersyncIfAmbiguous
 74       e.defaultTTLs = cfg.CacheURL.DefaultTTLs
 75 -     e.enforceCCPA = cfg.CCPA.Enforce
 76       return e
 77   }
exchange/exchange.go [RO]
```

Reply:
That's exactly the goal I'm working towards. Baby steps.